custom_components/netdaemon/const.py

```diff
@@ -6,7 +6,7 @@
 INTEGRATION_VERSION = "main"
 DOMAIN = "netdaemon"
 NAME = "NetDaemon"
-MINIMUM_HA_VERSION = "2020.12.0"
+MINIMUM_HA_VERSION = "2021.7.0"
```

Comment:
Not sure we want to make this dependency that soon?

Reply:
The select entity was added in 2021.7, when this is merged, it can only be used on 2021.7+

Reply:
@Laccolith thank you for this. I will merge it but wait a few versions until we are ready to force our users to 2021.7.